### PR TITLE
Update require_pulp_plugins

### PR DESCRIPTION
**Pull request by fao89**
[noissue]

### CODE REVIEW

**Comment by fao89** [comment]
@pulp/python the test were being skipped, it enables the tests again, but I don't have the background for fixing these tests

**Review by dralley — APPROVED** [review]

**Comment by fao89** [comment]
I rebased my PR, so I got this change:
https://github.com/pulp/pulp_python/pull/334/files
which uses http://pulp-fixtures:8080/
https://github.com/pulp/pulp_python/blob/master/.ci/ansible/smash-config.json#L34
but now the error is due https
what is the right fixture?

**Comment by gerrod3** [comment]
> what is the right fixture?

Bandersnatch requires the remote url to be https, so the fixtures url must point to our hosted fixtures and not use the container built fixtures.

**Comment by fao89** [comment]
> > what is the right fixture?
> 
> Bandersnatch requires the remote url to be https, so the fixtures url must point to our hosted fixtures and not use the container built fixtures.

if we set `docker-fixture: false` we will get `https://fixtures.pulpproject.org` but it may break CLI test:
https://github.com/pulp/plugin_template/commit/24fbd58ab19ea8f0ca6cb6173f55ac60c0d9ae6d
when you enable it:
https://github.com/pulp/pulp_python/blob/master/template_config.yml#L37

**Comment by gerrod3** [comment]
Hmm, where is that variable docker_fixtures being set? We should turn that off so that it chooses the hosted fixtures instead.

File: template_config.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ deploy_client_to_rubygems: true
 deploy_daily_client_to_pypi: true
 deploy_daily_client_to_rubygems: true
 deploy_to_pypi: true
-docker_fixtures: true
+docker_fixtures: false
```
Comment (fao89) [inline template_config.yml:17]: I turned it off and ran plugin-template

**Comment by fao89** [comment]
Finally green \o/
@pulp/python

**Pull request merged**